cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/model/AbstractApicurioConverter.java

```diff
@@ -9,14 +9,11 @@
 import org.bf2.cos.fleetshard.operator.debezium.DebeziumConstants;
 
 public abstract class AbstractApicurioConverter implements KafkaConnectConverter {
-
-    public static final String APICURIO_AUTH_SERVICE_URL = "https://identity.api.openshift.com/auth";
-
     @Override
     public Map<String, String> getAdditionalConfig(ManagedConnector config, ServiceAccountSpec serviceAccountSpec) {
         Map<String, String> additionalConfig = new HashMap<>();
-        additionalConfig.put("apicurio.auth.service.url", APICURIO_AUTH_SERVICE_URL);
-        additionalConfig.put("apicurio.auth.realm", "rhoas");
+        additionalConfig.put("apicurio.auth.service.url", DebeziumConstants.DEFAULT_APICURIO_AUTH_SERVICE_URL);
```

Comment:
Not sure but I believe in L15 and L16 we should point to the [new configs that were created](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/e802f04a8e803526b26484862f402fc31ee87413/cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandConfiguration.java#L15-L19)? Can you confirm @rk3rn3r?

Reply:
Exactly. @ani-sha please inject the `DebeziumOperandConfiguration` class or the URL and realm somehow here.